Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM quay.io/jupyter/minimal-notebook:afe30f0c9ad8
+
+COPY conda-linux-64.lock /tmp/conda-linux-64.lock
+RUN mamba update --quiet --file /tmp/conda-linux-64.lock \
+    && mamba clean --all -y -f \
+    && fix-permissions "${CONDA_DIR}" \
+    && fix-permissions "/home/${NB_USER}"
```

Comment:
Might need to automate the installation for `deepchecks` during the build process by adding `RUN pip install deepchecks==0.3.2` at the end of the `Dockerfile` (for `deepchecks` version, please refer to the comment on the `environment.yml` file to confirm which version to install).

Reply:
Is the `deepchecks` package not included on your end when you run the container? It seems to properly install on my end. Let me know, and if you have issues I will make sure to implement this.

Reply:
container runs fine on my end with `deepchecks` installed, thanks @bfrizzell01

Reply:
I've added the requested line to `Dockerfile`. Data Validation should now work properrly, @y1chi-z .